addons/website_sale/data/demo.xml

```diff
@@ -271,8 +271,7 @@
 
         <record id="list_christmas" model="product.pricelist">
             <field name="name">Christmas</field>
-            <field name="selectable" eval="False" />
-            <field name="website_id" ref="website.default_website" />
+            <field name="website_ids" eval="[(4, ref('website.default_website'))]"/>
```

Comment:
FYI, you can use Command.link in xml files too (it's available in the evaluation environment)